macros/AddonSidebar.ejs

```diff
@@ -325,7 +326,7 @@ var text = mdn.localStringMap({
     </li>
     <li class="toggle">
       <details>
-        <summary><%=text['WebExtensions#How_to']%></summary>
+       <summary><a href="<%=baseURL%>WebExtensions#How_to"><%=text['WebExtensions#How_to']%></a></summary>
```

Comment:
Is there a reason for this change? None of the other summaries have links, and it makes the sidebar look weird:

<img width="246" alt="screen shot 2018-10-25 at 8 34 52 pm" src="https://user-images.githubusercontent.com/432915/47543092-6faaee80-d895-11e8-9f4f-3b2ca8ea8d48.png">

Reply:
In correcting my merge error, I brought forward too much from the previous version.  Now removed.